src/schema/objects/metadata.yaml

```
@@ -896,6 +927,43 @@ EchoTime__fmap:
   type: number
   unit: s
   exclusiveMinimum: 0
+EditCondition:
+  name: EditCondition
+  display_name: Editing Condition
+  description: |
+    If spectral editing was applied, this lists the application order of `"EditPulse"`.
+    For example, `["ON", "OFF"]` for a MEGA-edited experiment or
+    `["A", "B", "C", "D"]` for a HERMES-edited experiment.
+  anyOf:
+    - type: string
+    - type: array
+      items:
+        type: string
+EditPulse:
+  name: EditPulse
+  display_name: Editing Pulse
+  description: |
+    If spectral editing was applied, this details the editing parameters.
+    For example:
+    `{"ON": {"FrequencyOffset": 1.9, "PulseDuration": 16},
+    "OFF": {"FrequencyOffset": 7.5, "PulseDuration": 16}}`.
+  anyOf:
+    - type: object
+      additionalProperties:
+        type: object
```

Here's a proposal that I believe should match the examples you have here. Please check whether you would want the fields to be required or recommended, and if I got the units correct.

```suggestion
  type: object
  additionalProperties:
    type: object
    recommended_fields: [FrequencyOffset, PulseDuration]
    properties:
      FrequencyOffset:
        anyOf:
          - type: number
            unit: MHz
          - type: array
            items:
              type: number
              unit: MHz
      PulseDuration:
        type: number
        unit: s
```

@rwblair could you check that my syntax is good?

---

For the units: `FrequencyOffset` should be `ppm` and `PulseDuration` should be `ms`.

---

Updated:

```suggestion
  type: object
  additionalProperties:
    type: object
    recommended_fields: [FrequencyOffset, PulseDuration]
    properties:
      FrequencyOffset:
        anyOf:
          - type: number
            unit: ppm
          - type: array
            items:
              type: number
              unit: ppm
      PulseDuration:
        type: number
        unit: ms
```